src/classes/config_file_reader.vala

```diff
@@ -82,6 +82,9 @@ namespace pdfpc {
                 bt.keyCode = keycode;
                 bt.modMask = modMask;
                 bt.actionName = fields[2];
+                if (fields.length > 3) {
+                    bt.actionArg = fields[3];
+                }
```

Comment:
This way, it is not a syntax error to pass an argument to an action that doesn't require one, right?